### PR TITLE
docs(inputs.ldap): Correct plugin name in sample config

### LINKED ISSUE #14445: LDAP docs show openldap as the input in example
### Relevant telegraf.conf

```toml
[[inputs.openldap]]
  server = "ldap://ldap.example.com"
  bind_dn = "uid=admin,cn=users,cn=accounts,dc=local,dc=example,dc=com"
  bind_password = "NOT_MY_P@$$W0rd"
```

### Logs from Telegraf

```text
2023-12-13T00:28:48Z I! Loading config: ldap.conf
2023-12-13T00:28:48Z E! error loading config file ldap.conf: plugin inputs.openldap: line 1: configuration specified the fields ["server"], but they weren't used
```

### System info

Telegraf 1.29, Fedora 39

### Docker

N/A

### Steps to reproduce

1. Open Relavent Docs ()
2. create config above
3. run telegraf --config ldap.conf --test --debug
...

### Expected behavior

making a config based on the docs should show results if ldap server is configured properly

### Actual behavior

input error shown above

### Additional info

changing the config to use `[[inputs.ldap]]` the issue is resolved

```
[[inputs.ldap]]
  server = "ldap://ldap.example.com"
  bind_dn = "uid=admin,cn=users,cn=accounts,dc=local,dc=example,dc=com"
  bind_password = "NOT_MY_P@$$W0rd"
```

### PR DESCRIPTION
**Pull request by tiny6996**
## Summary

Fixes bug with ldap configuration
## Checklist

- [x] No AI generated code was used in this PR

## Related issues

resolves #14445

### CODE REVIEW

File: plugins/inputs/ldap/README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
 
 ```toml @sample.conf
 # LDAP monitoring plugin
-[[inputs.openldap]]
+[[inputs.ldap]]
```
Comment (powersj) [inline plugins/inputs/ldap/README.md:20]: This is generated from the sample.conf file. You'll need to fix it in both to get tests to pass.

Thanks again!

**Review by powersj — APPROVED** [review]
Thanks again!

**Comment by tiny6996** [comment]
happy to help and thanks for the speedy review.
I'm assuming I don't need to do anything else and one of the maintainers will merge this?

**Comment by powersj** [comment]
>  I'm assuming I don't need to do anything else and one of the maintainers will merge this?

Correct! Even though the PR is small, we like both of us to sign off on PRs.

**Review by srebhan — APPROVED** [review]
Thanks for catching and correcting this @tiny6996!

**Pull request merged**